api/tests/integration/infrastructure/repositories/organization-learner-activity-repository_test.js

```diff
@@ -0,0 +1,196 @@
+      //given
+      const { id: organizationId } = databaseBuilder.factory.buildOrganization();
+      const { id: organizationLearnerId } = databaseBuilder.factory.buildOrganizationLearner({ organizationId });
+      const { id: otherOrganizationLearnerId } = databaseBuilder.factory.buildOrganizationLearner({ organizationId });
+
+      const { id: campaignId } = databaseBuilder.factory.buildCampaign({
+        organizationId,
+        name: 'other not so great campaign',
+        type: CampaignTypes.PROFILES_COLLECTION,
+      });
+
+      databaseBuilder.factory.buildCampaignParticipation({
+        campaignId,
+        organizationLearnerId,
+        createdAt: new Date('2000-01-01T10:00:00Z'),
+      });
+
+      databaseBuilder.factory.buildCampaignParticipation({
+        campaignId,
+        otherOrganizationLearnerId,
+        createdAt: new Date('2005-01-01T10:00:00Z'),
+      });
+
+      await databaseBuilder.commit();
+
+      //when
+      const {
+        participations: [organizationLearnerParticipation],
+      } = await organizationLearnerActivityRepository.get(organizationLearnerId);
+
+      //then
+      expect(organizationLearnerParticipation.createdAt).to.deep.equal(new Date('2000-01-01T10:00:00Z'));
+    });
+
+    it('Should return an activity with the participations in anti-chronological order', async function () {
+      //given
+      const { id: organizationLearnerId, organizationId } = databaseBuilder.factory.buildOrganizationLearner();
+
+      const { id: firstCampaignId } = databaseBuilder.factory.buildCampaign({
+        organizationId,
+        name: 'Aurelies super campaign',
+        type: CampaignTypes.PROFILES_COLLECTION,
+      });
+
+      const { id: secondCampaignId } = databaseBuilder.factory.buildCampaign({
+        organizationId,
+        name: 'Karams super campaign',
+        type: CampaignTypes.ASSESSMENT,
+      });
+
+      databaseBuilder.factory.buildCampaignParticipation({
+        campaignId: firstCampaignId,
+        organizationLearnerId,
+        status: SHARED,
+        createdAt: new Date('2000-01-01T10:00:00Z'),
+        sharedAt: new Date('2000-01-02T10:00:00Z'),
+      });
+
+      databaseBuilder.factory.buildCampaignParticipation({
+        campaignId: secondCampaignId,
+        organizationLearnerId,
+        status: STARTED,
+        createdAt: new Date('2000-01-03T10:00:00Z'),
+      });
+
+      await databaseBuilder.commit();
+
+      //when
+      const {
+        participations: [secondOrganizationLearnerParticipation, firstOrganizationLearnerParticipation],
+      } = await organizationLearnerActivityRepository.get(organizationLearnerId);
+
+      //then
+      expect(secondOrganizationLearnerParticipation.createdAt).to.deep.equal(new Date('2000-01-03T10:00:00Z'));
+      expect(firstOrganizationLearnerParticipation.createdAt).to.deep.equal(new Date('2000-01-01T10:00:00Z'));
+    });
+    describe('campaign type in participation', function () {
```

Comment:
**suggestion**: Les tests sur le type de campagne et le status de la participation n'ont pas de plus value par rapport au test Should return an activity with all attributes of existing participations. De plus ils deviennent obsolète si on ajoute un type de campagne ou un status de participation.